packages/store/src/storage.ts

```diff
@@ -1405,6 +1405,9 @@ class Storage {
               throw errorUtils.getBadRequest(errorMessage);
             }
           }
+          if (tarball === '__proto__' || tarball === 'constructor' || tarball === 'prototype') {
```

Comment:
I think there is a file name checkup at middleware level but :) never is late to check again

Reply:
true. code scan isn't that smart. getting rid of warnings makes it more obvious if new warnings appear, though 😀